src/relion/zocalo/tomo_align_iris.py

```diff
@@ -0,0 +1,615 @@
+from __future__ import annotations
+
+import ast
+import os
+import os.path
+import time
+from pathlib import Path
+from typing import List, Optional, Union
+
+import htcondor
+import plotly.express as px
+import procrunner
+import workflows.recipe
+import workflows.transport
+from pydantic import BaseModel, Field, validator
+from pydantic.error_wrappers import ValidationError
+from workflows.services.common_service import CommonService
+
+
+class TomoParameters(BaseModel):
```

Comment:
Is this class the same as the one in `tomo_align.py`? If it is it could be imported rather than having a repeat of the code